src/Card/CardImageCap.jsx

```diff
@@ -10,9 +11,17 @@ const CardImageCap = React.forwardRef(({
   logoAlt,
   className,
 }, ref) => {
-  const { orientation } = useContext(CardContext);
+  const { orientation, isLoading } = useContext(CardContext);
   const wrapperClassName = `pgn__card-wrapper-image-cap ${orientation}`;
 
+  if (isLoading) {
+    return (
+      <div className={classNames(className, wrapperClassName)}>
+        <Skeleton containerClassName="pgn__card-image-cap-loader" height="100%" />
```

Comment:
When providing images to the `CardImageCap` component, consumers can provide both the primary background image but also a logo image. I think the loading state for this component should include support for skeletons both in the main background image but also within the small rectangle for the logo image to more accurately reflect the content that will be returned.

<img width="562" alt="image" src="https://user-images.githubusercontent.com/2828721/175466641-ec2e99ff-1d5e-475f-922b-fc88804eb724.png">

Perhaps there's an extra prop on `Card.ImageCap` such as `logoSkeleton` that could control whether the consumer wants to display a skeleton where the logo image will ultimately go when `isLoading=true`? I think it'd have to be an extra prop the consumer has to pass versus being able to recognize programmatically when to display the skeleton for the logo.

What do you think?

Reply:
Yes, good idea. It's very flexible for user